pkg/projects/Microsoft.NETCore.App/netcoreapp1.1/project.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
We shouldn't need to reference Targets or the new System.Runtime. you should instead reference the new NETStandard.Library package which contains both of those. 

Actually I think we can get away with a single package list for both netcoreapp1.0 and netcoreapp1.1 and just update the versions. The asset selection isn't based on the version but based on the contents of the actual packages so everything should just work out.